addon/lib/metrics.js

```diff
@@ -65,90 +59,70 @@ const variantMaker = {
 const Metrics = module.exports = {
 
   init: function() {
-    if (!store.telemetryPingPayload) {
-      store.lastTelemetryPingTimestamp = false;
-      store.telemetryPingPayload = {
-        version: 1,
-        tests: {}
-      };
-    }
-
-    Metrics.maybePingTelemetry();
-
+    store.browserLoadedTimestamp = Date.now();
```

Comment:
So, we're supposed to record times based on when the browser was loaded. I'm not sure that's possible from an addon; the best I was able to figure out is the amount of time since the add-on was loaded, which will be roughly equivalent in most cases, but if the user installed the testpilot addon since they launched the browser, it will be the amount of time since that event.

Thoughts? @ilanasegall @rjweiss